actions/command.ts

```diff
@@ -111,6 +115,16 @@ export function executeCommand(message: string, args: CommandArgs): ActionFunc {
             dispatch(openModal({modalId: ModalIdentifiers.USER_SETTINGS, dialogType: UserSettingsModal, dialogProps: {isContentProductSettings: true}}));
             return {data: true};
         case '/marketplace':
+            // check if user has permissions to access the read plugins
+            if (!haveICurrentTeamPermission(state, Permissions.SYSCONSOLE_READ_PLUGINS)) {
+                return {error: {message: localizeMessage('marketplace_command.no_permission', 'You do not have the appropriate permissions to access the marketplace')}};
+            }
+
+            // check config to see if marketplace is enabled
+            if (!isMarketplaceEnabled(state)) {
```

Comment:
nit: Maybe check first the marketplace enabled?

Reply:
I did that first but realized that a non-admin user has no business knowing what's the status of the configuration so it made more sense to make this check first